runtime.ts

```diff
@@ -1176,15 +1176,26 @@ module J2ME {
   }
 
   function findNativeMethodImplementation(methodInfo: MethodInfo) {
+    function wrap(method, key) {
```

Comment:
Nit: I would call this something like reportError to make its purpose clearer.

Also, we should declare this outside the findNativeMethodImplementation function, so we don't have to redeclare it each time that function is called.